frontend/packages/local-storage-operator-plugin/src/components/local-volume-discovery/body.tsx

```diff
@@ -17,6 +19,11 @@ export const LocalVolumeDiscoveryBody: React.FC<LocalVolumeDiscoveryBodyProps> =
   taintsFilter,
 }) => {
   const { t } = useTranslation();
+  const [discoveredNodes] = useK8sWatchResource<NodeKind[]>(lvdrResource);
+  const preSelectedNodes = [
+    ...selectNodes.map(getName),
+    ...discoveredNodes.map((node) => getLabel(node, 'discovery-result-node')),
```

Comment:
This is more reliable
```suggestion
    ...lvdrList.map((node) => lvdrList.spec.nodeName,
```